include/mbgl/ios/MGLGeometry.h

```diff
@@ -1,10 +1,11 @@
 #pragma once
 
-#import "MGLTypes.h"
```

Comment:
This breaks compilation on Xcode 6.2, because MGLTypes.h is responsible for defining `NS_ASSUME_NONNULL_BEGIN` et al. on older Xcodes. I don’t know if we still want to support 6.2 anymore.